lib/rpm_package.py

```diff
@@ -187,19 +187,25 @@ def _load(self):
                 self.build_files = None
             self.download_build_files = files.get('download_build_files', [])
 
+
+            rebuild_dependencies = CONF.get('default').get('force_rebuild')
```

Comment:
This option had a slightly different meaning in a previous version of the PR.
Changing to `self.rebuild`